tests/helpers/util.py

```diff
@@ -380,6 +380,10 @@ def get_container_file_content(container, path):
     return container.exec_run("cat %s" % path)[1].decode("utf-8")
 
 
+def get_stripped_container_id(container_id):
+    return re.sub(r"^[\w_-]+://", "", container_id)
```

Comment:
This is a test anti-pattern by implementing the test logic identical to the app logic.  If you happened to have a mistake in the regexp, you would never know it.  Make this something more explicit with `.replace` like it was originally (but still keep the separate function so we can add more easily later).

Reply:
Ah good point, will update